class/frontend-form-post.php

```diff
@@ -508,7 +508,9 @@ function submit_post() {
 
                 // if mail active
                 if ( isset( $user_wpuf_subscription_pack['_enable_mail_after_expired'] ) && $user_wpuf_subscription_pack['_enable_mail_after_expired'] == 'on' ) {
-                    $post_expiration_message = $user_wpuf_subscription_pack['_post_expiration_message'];
+                    $wpuf_user  = wpuf_get_user();
+                    $user_subscription = new WPUF_User_Subscription( $wpuf_user );
+                    $post_expiration_message = $user_subscription->get_subscription_exp_msg( $user_wpuf_subscription_pack['pack_id'] );
```

Comment:
For consistency we should use the pack_id from $user_subscription instead of using the older $user_wpuf_subscription_pack var